### DISCUSSION TITLE
Welcome to wdio-video-reporter Discussions!

### DISCUSSION THREAD

**Discussion by presidenten** [webdriverio-community/wdio-video-reporter#50] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by learnautomatedtesting** [comment]
hi I need some support with a boiler plate project with supporting ESM with typescript and video reporter. Can you support me on this. I can make my example project availabe in github if you would like .I tried a lot today, but somehow it gets stuck on starting the browser. btw I am a newbee a bit in typescript still

**Reply by christian-bromann** [reply]
I recommend to check out the demo directory we have in this repo: https://github.com/webdriverio-community/wdio-video-reporter/tree/main/demo

**Reply by learnautomatedtesting** [reply]
thank you, I will check it out
